2-Operating-Systems/1-arch.md

```diff
@@ -198,7 +202,7 @@ Arch Linux. Power users with different opinions should refer to the official
 
   ```
   # Create the hostname file
-  echo "computername" > /etc/hostname
+  hostnamectl set-hostname computername
```

Comment:
you only need this hostnamectl command. the change to /etc/hosts isn't required as nss-myhostname is a thing since 2013 or so :)